common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -115,112 +90,157 @@ OSD Panel Items
+ - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 33 (DJI)
+ - :ref:`SERIAL2_BAUD<SERIAL2_BAUD>` = 115
+ - :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 0 = 0 (Do not EnableTelemetryMode)
 
-Displaying statistics on a dedicated screen requires enabling at least one extra screen by setting the respective ``OSDn_ENABLE`` to 1.
-By default, ArduPilot has only one screen active so in a typical setup one would set (:ref:`OSD2_ENABLE<OSD2_ENABLE>`) = 1 and then enabling the OSD stats panel on screen 2 by setting (:ref:`OSD2_STATS_EN<OSD2_STATS_EN>`) = 1.
+.. note:: Serial port buad rate default is changed to 115.2Kbaud automatically when setting the above protocol type. However, if the user has previously or subsequently changes the baud, this default will not be used. 115.2Kbaud is required by most video goggle systems.
 
-When the OSD switches to this screen it will check the value of the :ref:`OSD2_STATS_EN<OSD2_STATS_EN>` parameter and if enabled it will override the default behavior of the following OSD items:
+.. note:: DJI Custom OSD must be enabled: in SETTINGS->DISPLAY->CUSTOM OSD menu of goggles
 
- - OSDn_MESSAGE will display STATS followed by flight time
- - OSDn_ALTITUDE will display max altitude
- - OSDn_BAT_VOLT will display min voltage
- - OSDn_CURRENT will display max current
- - OSDn_GSPEED will display max ground speed (or airspeed if ``OSDn_ASPEED_EN`` is set to 1)
- - OSDn_HOMEDIST will alternates max distance from home and total traveled distance every 2 seconds
- - OSDn_RSSI will display min rssi
+By setting :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 2 to 1 (value = 4) one can force ArduPilot to impersonate Betaflight and use a Betaflight compatible font index for the font table integrated in the remote OSD system. This is only required if the remote OSD system does not have an ArduPilot compatible fonts table (such as DJI O3 systems and non-wtfos converted DJI goggles).
 
-DisplayPort OSD
-===============
+.. note:: the direction arrows will be reversed since ArduPilot and Betaflight use direction arrows in their font tables that are 180 deg different than each other. This can be corrected by using :ref:`OSD_OPTIONS<OSD_OPTIONS>` bit 5 to invert them before sending to OSD.
 
-HDZero using ArduPilot custom fonts
+MWOSD, Walksnail, and DJI goggles using the wtf-os/msp-osd firmware already support custom fonts locally, and HDZero has an ArduPilot compatible font set.
 
-.. image:: ../../../images/msp_osd_displayport.jpg
-   :target: ../_images/msp_osd_displayport.jpg
+Default behavior (:ref:`MSP_OPTIONS<MSP_OPTIONS>` Bit 2 = 0) is to use the ArduPilot font table's indexes.
```

Comment:
This discussion of fonts now reads like it only applies to DisplayPort. I'm going to suggest a change which moves it down into the OSD section.